libs/api-hjarntorget/lib/apiHjarntorget.ts

```diff
@@ -571,6 +572,22 @@ export class ApiHjarntorget extends EventEmitter implements Api {
     return statusChecker
   }
 
+  public async registerAbscense(child: EtjanstChild, startDate: Date, endDate: Date): Promise<void> {
+
+      const body = {
+        attendeeId: child.id,
+        startDate: startDate,
+        endDate: endDate,
```

Comment:
The date needs to be formatted like this in the body: `2021-11-25+00%3A00`. Here the dates get stringified to `Thu Nov 25 2021 00:00:00 GMT+0100 (Central European Standard Time)`